README.md

```diff
@@ -1 +1,28 @@
-# Conduit_Container
\ No newline at end of file
+# Conduit_Container
+
+A Docker Compose container that builds two containers, a Conduit backend and a Conduit frontend.
+
+## Requirements
+
+- Docker Compose
```

Comment:
missing requirements:

- Docker
- Basic Knowledge about container definition
- basic knowledge about shell-scripting, python, and linux